src/assets/data/integrations/IntegrationList.js

```diff
@@ -196,6 +199,24 @@ const IntegrationList = [
     status: "inProduction",
     category: "Service Mesh",
   },
+  {
+    name: "Prometheus",
+    img: prometheus,
+    status: "inProduction",
```

Comment:
All these integrations are in the in-progress state to change the status to InProgress.